test/libsolidity/semanticTests/array/copying/copy_function_storage_array.sol

```diff
@@ -18,6 +18,6 @@ contract C {
 // compileViaYul: also
 // ----
 // test() -> 7
-// gas irOptimized: 134482
+// gas irOptimized: 134680
```

Comment:
Here is the diff: https://gist.github.com/hrkrshnn/d6db937fce08b059f7339b8f36f93309

The loop condition to body is not getting reverted. Quite surprising.

Reply:
Interesting. Also the constant is not pulled out of the loop.